src/docs/customize/global-props.md

```diff
@@ -83,3 +83,13 @@ React.createElement(() => {
   );
 });
 ```
+
+## Nested Usage
```

Comment:
I was thinking about the code example. But in the end, I decided to omit it as it seems quite straighforward to me.

Reply:
I would prefer to include it to illustrate scalar, object and array handling.

Reply:
Added example that contains adding new prop, overriding and removing. Grid is only component where objects can be logically used. We do not have any array as a prop so it cannot be documented.